src/index.js

```diff
@@ -3,14 +3,23 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import './index.scss';
 import reportWebVitals from './reportWebVitals';
+import { i18n } from '@lingui/core';
+import { I18nProvider } from '@lingui/react';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import store from './store/store';
+import catalogEn from '@lingui/loader!./locales/en/messages.po';
+import catalogRu from '@lingui/loader!./locales/ru/messages.po';
+i18n.load('en', catalogEn.messages);
+i18n.load('ru', catalogRu.messages);
+i18n.activate('en');
```

Comment:
make a dynamic loading based on localstorage. You can create a langService or king of and move all logic there